config/locales/devise_invitable.en.yml

```diff
@@ -0,0 +1,31 @@
+en:
```

Comment:
I'd like to see this file go away, since we have its contents in `config/locales/en.yml`.